scenes/game/local_game.gd

```diff
@@ -4156,6 +4155,9 @@ func handle_strike_effect(card_id :int, effect, performing_player : Player):
 			_append_log_full(Enums.LogType.LogType_Effect, performing_player, "'s %s is no longer blocking opponent movement." % buddy_name)
 		"return_attack_to_top_of_deck":
 			performing_player.strike_stat_boosts.attack_to_topdeck_on_cleanup = true
+		"nothing":
```

Comment:
Let's just leave this out and use pass? I think there was just that one effect using nothing

Reply:
oh wait, its in the strings, disregard